1-js/02-first-steps/01-hello-world/1-hello-alert/solution.view/index.html

```diff
@@ -4,9 +4,9 @@
 <body>
 
   <script>
-    alert( "I'm JavaScript!" );
+    alert( "Мен JavaScript!" );
```

Comment:
Мүмкін осылай аударылу керек?
```suggestion
    alert( "Мен JavaScript-пін!" );
```
```suggestion
    alert( "Мен JavaScript-пін!" );
```